swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -186,43 +200,39 @@ service /healthcare on new http:Listener(port) {
 
 The [sequence diagram view](https://wso2.com/ballerina/vscode/docs/implement-the-code/sequence-diagram-view/) for the implemented resource method is the following.
 
-<img src="/learn/images/tutorial_sending_a_message_to_a_service.png" alt="Sequence Diagram" height="800" style="width:auto; max-width:100%">
+<img src="/learn/images/integration-tutorials/sending-a-message-to-a-service/sequence_diagram.png" alt="Sequence Diagram" height="800" style="width:auto; max-width:100%">
 
 ### Step 3: Build and run the service
 
-You can run this service by navigating to the project root and using the `bal run` command.
+![Run the service](/learn/images/integration-tutorials/sending-a-message-to-a-service/run_the_service.gif)
 
-```bash
-sending-a-message-to-a-service$ bal run
-Compiling source
-        integration_tutorials/sending_a_message_to_a_service:0.1.0
-
-Running executable
-```
+> **Note:**
+> Alternatively, you can run this service by navigating to the project root and using the `bal run` command.
+>
+> ```bash
+> sending-a-message-to-a-service$ bal run
+> Compiling source
+>         integration_tutorials/sending_a_message_to_a_service:0.1.0
+>
+> Running executable
+> ```
+>
 
 ### Step 4: Try out the use case
 
 Let's test the use case by sending a request to the service.
 
 #### Start the backend service
 
-Download the JAR file for the [backend service](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/hospitalservice.jar), and execute the following command to start the service:
+Download the JAR file for the [backend service](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/hospitalservice.jar), and execute the following command to start the service.
 
 ```bash
 bal run hospitalservice.jar
 ```
 
 #### Send a request
 
-Let's send a request to the service using cURL as follows.
-
-1. Install and set up [cURL](https://curl.se/) as your client.
-
-2. Execute the following command.
-
-    ```bash
-    curl -v http://localhost:8290/healthcare/querydoctor/surgery
-    ```
+![Send a request](/learn/images/integration-tutorials/sending-a-message-to-a-service/try_it.gif)
```

Comment:
Can we mention something about the try it client here. Basically a simple description of the step.